### DISCUSSION TITLE
Welcome to stumpwm Discussions!

### DISCUSSION THREAD

**Discussion by dmb2** [stumpwm/stumpwm#1105] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by funk443** [comment]
Hello, I am just an average Lisp enjoyer.

I wonder if we can add `WM_CLASS` by default for modeline, message windows, and other windows made by StumpWM itself, so it is easier for users to add compositor exclude rules for these windows.

I currently have these codes to do this:
```
(xlib:set-wm-class (screen-message-window (current-screen))
                   "stumpwm-message" "stumpwm-message")

(add-hook *new-mode-line-hook* (lambda (mode-line)
                                 (xlib:set-wm-class (mode-line-window mode-line)
                                                    "stumpwm-modeline"
                                                    "stumpwm-modeline")))
```
